nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt

```diff
@@ -168,29 +122,25 @@ licenses donated by YourKit), etc.
 
 [1]: http://www.nuxeo.com/downloads/
 [2]: http://nuxeo.github.com/downloads.html
-[3]: http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html
+[3]: http://www.oracle.com/technetwork/java/javase/downloads/index.html
 [4]: http://openjdk.java.net/install/index.html
-[5]: https://doc.nuxeo.com/x/ngQz
-[6]: https://doc.nuxeo.com/x/G4AO
-[7]: http://doc.nuxeo.com/x/NIpH
-[8]: http://doc.nuxeo.com/x/6QJc
+[5]: https://doc.nuxeo.com/nxdoc/adding-an-antivirus/
+[7]: https://doc.nuxeo.com/nxdoc/configuring-the-nuxeo-platform-as-a-daemon-with-sysvinit/
+[8]: https://doc.nuxeo.com/nxdoc/installing-the-nuxeo-platform-as-a-windows-service/
 [9]: http://localhost:8080/nuxeo/nxhome/default@view_admin?tabIds=NUXEO_ADMIN%3AConnectApps
 [10]: https://marketplace.nuxeo.com/
-[11]: https://doc.nuxeo.com/x/PwA7
+[11]: https://doc.nuxeo.com/nxdoc/administration/
 [17]: https://download.nuxeo.com/browser/firefox/nuxeo-dragdrop-ff-extension.xpi
 [18]: http://download.nuxeo.org/desktop-integration/drag-drop/msie/
```

Comment:
not used anymore

Reply:
Actually there is still a page with various links.